### PR TITLE
Going through French grammar

### PR DESCRIPTION
**Pull request by deatheguard**
Fixed the grammar and added appropriate French analogues in the beginning.
Will continue later. :)

### CODE REVIEW

File: php/Locale/French.php [hunk 1]
```diff
@@ -14,17 +14,17 @@ static public function get_locale()
         $locale	= array(
             'USERNAME'		=> 'Nom d\'utilisateur',
             'PASSWORD'		=> 'Mot de passe',
-            'ENTER'			=> 'Se connecter',
+            'ENTER'			=> 'Enter',
```
Comment (ildyria) [inline php/Locale/French.php:17]: Actually should be 'Entrer' my mistake...

File: php/Locale/French.php [hunk 2]
```diff
@@ -79,30 +79,30 @@ static public function get_locale()
...
             'UNSORTED'					=> 'Non-triés',
-            'STARED'					=> 'Etoiles',
+            'STARED'					=> 'Favoris',
             'RECENT'					=> 'Récent',
             'PUBLIC'					=> 'Public',
             'NUM_PHOTOS'				=> 'Photos',
 
             'CREATE_ALBUM'				=> 'Créer un album',
 
-            'STAR_PHOTO'				=> 'Favoriser la photo',
```
Comment (ildyria) [inline php/Locale/French.php]: Favoriser does not have that meaning in French: it means that you have a preference for X and you are doing something for that X. In our case here, we are placing something in our `Favoris`.

File: php/Locale/French.php [hunk 3]
```diff
@@ -79,30 +79,30 @@ static public function get_locale()
...
-            'TAGS_ALL'					=> 'Identifier tout',
-            'UNSTAR_PHOTO'				=> 'Défavoriser la photo',
+            'STAR_PHOTO'				=> 'Metre en Favoris',
+            'STAR'						=> 'Favori',
+            'STAR_ALL'					=> 'Tout mettre en Favoris',
+            'TAGS'						=> 'Tagger',
+            'TAGS_ALL'					=> 'Tout tagger',
+            'UNSTAR_PHOTO'				=> 'Retirer des Favoris',
 
-            'FULL_PHOTO'				=> 'Dimension réelle',
```
Comment (ildyria) [inline php/Locale/French.php]: While `Dimension` and `Taille` have meaning pretty close, in this context `Taille` is better as it also provide the idea of size of the picture (MB) while `Dimension` is more a spacial *size*.

File: php/Locale/French.php [hunk 4]
```diff
@@ -147,16 +147,16 @@ static public function get_locale()
             'PHOTO_NEW_TAGS_1'			=> 'Entrez vos tags pour toutes les',
             'PHOTO_NEW_TAGS_2'			=> 'photos selectionnées. Les tags existants seront remplacés. Vous pouvez ajouter plusieurs tags en les séparant avec une virgule:',
             'PHOTO_SET_TAGS'			=> 'Etablir les tags',
-            'PHOTO_CAMERA'				=> 'Appareil photographique',
```
Comment (ildyria) [inline php/Locale/French.php]: We are here in a context of camera, no need to say that. It is like saying `Device` and `Camera device`.

File: php/Locale/French.php [hunk 5]
```diff
@@ -147,16 +147,16 @@ static public function get_locale()
...
+            'PHOTO_CAMERA'				=> 'Appareil',
             'PHOTO_CAPTURED'			=> 'Date d\'enregistrement',
             'PHOTO_MAKE'				=> 'Prendre une photo',
             'PHOTO_TYPE'				=> 'Type',
-            'PHOTO_SHUTTER'				=> 'Obturateur',
```
Comment (ildyria) [inline php/Locale/French.php]: `Obturateur` is the physical part of the camera. Here we want it's speed / length of exposure.

**Comment by ildyria** [comment]
@deatheguard I updated your propositions with some comments of why I made these choices as a native French speaker.

**Comment by deatheguard** [comment]
> @deatheguard I updated your propositions with some comments of why I made these choices as a native French speaker.

thank you! There's always room for my language improvement. 
I will continue reviewing your project and fixing grammar as soon as I can! (probably tomorrow)

**Pull request merged**